xml/ha_hawk2_health_i.xml

```diff
@@ -26,8 +26,9 @@
  <para>
   &hawk2; provides a wizard which checks and detects issues with your cluster.
   After the analysis is complete, &hawk2; creates a cluster report with further
-  details. To verify your cluster, &hawk2; requires SSH access between your
-  cluster nodes.
+  details. To verify cluster health and generate the report, &hawk2; requires
+  passwordless SSH access between the nodes, otherwise it can only collect data
+  from the current node.
```

Comment:
Maybe adding a note that the cluster bootstrap commands automatically configure this access here would be good?